src/ml/neural_net/mps_graph_cnnmodule.mm

```diff
@@ -163,7 +164,7 @@ - (instancetype)initWithCommandBuffer:(id<MTLCommandBuffer>)commandBuffer {
       std::vector<float> loss(loss_size);
       auto loss_it = loss.begin();
       for (MPSCNNLossLabels *lossState in batch.lossState) {
-        *loss_it = sumImage([lossState lossImage]);
+        *loss_it = sumImage([lossState lossImage]) / MPS_LOSS_MULTIPLIER;
```

Comment:
Could you add an explanatory comment either here or where the constant is defined?